api/src/main/resources/db/migration/V10__CreateUsersTable.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS Users (
```

Comment:
This file is a duplicate of a file with the same name, but a `V9` prefix.  We don't need both.  However, there's also already another file with the `V9` prefix, so perhaps this one should live on and the other be removed.